charts/alertmanager/values.yaml

```diff
@@ -29,7 +29,32 @@ replicas: 1
 
 dataDir: "/data"
 
-prefixURL: ""
+readinessProbe:
+  httpGet:
+    path: "/#/status"
+    port: 9093
+  initialDelaySeconds: 30
+  timeoutSeconds: 30
+
+livenessProbe: {}
+
+# This section is templated in order to allow us to embed helm values from elsewhere
+authProxy:
+  livenessProbe:
+    httpGet:
+      path: /healthz
+      port: "{{ .Values.global.authSidecar.port }}"
+      scheme: HTTP
+    initialDelaySeconds: 10
+    periodSeconds: 10
+  readinessProbe:
+    httpGet:
+      path: /healthz
+      port: "{{ .Values.global.authSidecar.port }}"
+      scheme: HTTP
+    initialDelaySeconds: 10
+    periodSeconds: 10
```

Comment:
This section is run through a `{{ tpl (toYaml ...) . | nindent 12 }}` section so that the helm values from global are rendered in the k8s manifest output.